client/css/style.css

```diff
@@ -603,11 +603,6 @@ button {
 #chat .msg:last-child .text {
 	padding-bottom: 10px;
 }
-#chat .msg .type {
-	color: #ccc;
-	display: none;
-	font-style: normal;
-}
```

Comment:
As you can see in the `msg` template, `<em class="type">{{type}}</em>` was removed as it's not needed anymore, it was used to display the action.